lib/sequent/generator/command.rb

```diff
@@ -76,6 +76,24 @@ def ensure_existing_aggregate!
           raise NoAggregateFound
         end
       end
+
+      def command_template
+        ERB.new <<~EOF
+
+          class <%= command %> < Sequent::Command
```

Comment:
I would prefer this in separate ERB files, more common and more readable. Or is there a particular reason not to use an ERB file?